darwin/bin/build_pack.sh

```diff
@@ -41,7 +41,7 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
 fi
 
 electron-packager . \
-  --executable-name headset \
+  --executable-name Headset \
```

Comment:
Why are we changing this again?

Reply:
I didn't realize it earlier, but for mac, the executable (App Name) needs to be title case. there are not hard rules around it, just a convention.

Reply:
The .app looks like this, where it should be `Headset`
![Uploading Screen Shot 2017-12-26 at 1.43.08 PM.png…]()